UPGRADE.md

```diff
@@ -56,13 +56,28 @@ In the past, Gregorio handled the notes of an `<eu>` block like any other, which
 
 #### New algorithm
 
-A new algorithm has been implemented but is turned off by default. We hope this will be a significant improvement over previous one, but it still has a few bugs. You can turn it on with
+A new algorithm has been implemented but is turned on by default.  We hope this will be a significant improvement over the previous one.  The goal of the new algorithm is to place the bar line exactly between the notes which surround it and do the same with the text associated with the bar line while at the same time keeping the text and bar reasonably close together and with sufficent space between the bar syallable and the surrounding syllables (possibly introducing space to meet the competing interests).  The new algorithm introduces sevearl new spaces allowing the user to tune the algorithm:
```

Comment:
- `but is turned on by default` should probably be `and is turned on by default` or `and is used by default`.
- `sevearl` should be `several`
- The penultimate sentence almost feels like a run-on sentence.  I think it needs to be broken up into several sentences, or perhaps some sort of punctuation should be added.